src/main/java/de/unistuttgart/iste/gits/skilllevel_service/service/calculation/SkillLevelCalculator.java

```diff
@@ -287,4 +295,24 @@ public AssessmentRepetition(OffsetDateTime timestamp, float earnedSkillPoints) {
             this.earnedSkillPoints = earnedSkillPoints;
         }
     }
+
+    /**
+     * Helper class which can be used to track skill points of all types. Should not be used outside of this
+     * class because it has no encapsulation. Only used inside this parent class to make calculation methods more tidy.
+     */
+    private static class AllSkillPoints {
```

Comment:
dA hÄtTe MaN jEtZt ScHoN eIn ReCoRd NeHmEn KöNnEn

Reply:
> dA hÄtTe MaN jEtZt ScHoN eIn ReCoRd NeHmEn KöNnEn

Ne weil bei nem record wären die felder final.

Ich brauch halt die Möglichkeit die zu modifizieren. Ich weiß dass SQ rummeckert aber für ne *interne* (private) Klasse finde ich das in Ordnung. Die dient ja nur dafür, dass ich nicht meine Methoden mit 4mal so vielen Variablen vollspammen muss weil ich eine für jeden Skill Type bräuchte